### PR TITLE
Add support for environment variables using the %env(VAR)% symfony syntax

### PR DESCRIPTION
**Pull request by nocive**

### CODE REVIEW

File: tests/ConfigServiceProviderTest.php [hunk 1]
```diff
@@ -103,6 +103,39 @@ public function testRegister()
...
+        $adapter = $this->createMock(ConfigAdapter::class);
+
+        $adapter->method('load')->willReturn([
+            'env.var' => '%env(ENV_VAR)%',
+            'env.var.1' => '%env(ENV_VAR_1)%',
+            'envvar' => '%env(ENVVAR)%',
+            'envvar1' => '%env(ENVVAR1)%',
+        ]);
+
+        putenv('ENV_VAR', 'foo');
```
Comment (misantron) [inline tests/ConfigServiceProviderTest.php]: should be `putenv('ENV_VAR=foo')`

**Review by misantron — COMMENTED** [review]
Fix the test please

File: tests/ConfigServiceProviderTest.php (hunk 1 above)
Comment (nocive) [reply tests/ConfigServiceProviderTest.php]: thanks, fixed!

**Comment by nocive** [comment]
thanks once again! :hand:

**Pull request merged**